requirements/test.txt

```diff
@@ -4,3 +4,4 @@ pytest
 pytest-django
 pytest-cov
 coverage
+tox
```

Comment:
So how is it supposed to be listed as a req't for the project? Should devs just be responsible for installing it themselves?